source/manual/restore-from-offsite-backups.html.md

```diff
@@ -27,6 +27,16 @@ on a Vagrant VM.
 
 On a fresh VM, you may require the following packages for this exercise:
 
+> You can use either your dev VM or if you have the space you can create a new new mysql server VM using the following command:
+>
+> ```vagrant up mysql-master-1.backend```
+>
+> This needs to be run from the `/govuk_puppet` directory not the `/govuk_puppet/development_vm`
```

Comment:
I would leave the end off of this sentence:

`This needs to be run from the root of govuk-puppet repository`.

Reply:
good call